Makefile

```diff
@@ -571,7 +574,18 @@ generate-metrics-config: $(ENVSUBST_BIN) ## Generate ./hack/observability/kube-s
 
 .PHONY: generate-diagrams
 generate-diagrams: ## Generate diagrams for *.plantuml files
-	$(MAKE) -C docs diagrams
+	$(MAKE) generate-diagrams-book
+	$(MAKE) generate-diagrams-proposals
+
+.PHONY: generate-diagrams-book
+generate-diagrams-book: ## Generate diagrams for *.plantuml files in book
+	-docker run -v $(ROOT_DIR)/$(DOCS_DIR):/$(DOCS_DIR)$(DOCKER_VOL_OPTS) plantuml/plantuml:$(PLANTUML_VER) /$(DOCS_DIR)/book/**/*.md
```

Comment:
What is the - prefix doing?

Reply:
Honestly don't know - didn't see it when I was moving over the targets. Removed now

Reply:
Got lucky via google: https://stackoverflow.com/questions/3477292/what-do-and-do-as-prefixes-to-recipe-lines-in-make

> - means ignore the exit status of the command that is executed (normally, a non-zero exit status would stop that part of the build).

Sounds reasonable to drop it